tools/check_symbols.py

```
@@ -0,0 +1,66 @@
+    ("random_buffer", "core/embed/sys/rng/stm32/rng.c"),
+]
+
+class Error(Exception):
+    def __init__(self, line: str) -> None:
+        super().__init__(line)
+
+
+def _check(elf_file: Path) -> None:
+    click.echo(f"🔵 {elf_file}")
+    args = ["arm-none-eabi-nm", "--line-numbers", "-C", elf_file]
+    lines = sp.check_output(args).decode().splitlines()
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)